src/main/java/lv/ctco/springboottemplate/features/greeting/GreetingService.java

```diff
@@ -0,0 +1,18 @@
+package lv.ctco.springboottemplate.features.greeting;
+
+import lv.ctco.springboottemplate.features.todo.TodoService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class GreetingService {
+    private final TodoService todoService;
+
+    public GreetingService(TodoService todoService) {
+        this.todoService = todoService;
+    }
+
+    public String greet() {
+        var todosCount = todoService.getAllTodos().stream().filter(todo -> !todo.completed()).count();
+        return String.format("Hello from Spring! You have %s open tasks.", todosCount);
+    }
+}

```

Comment:
“tasks” plural should only be used when there is not a single task in the system, use singular otherwise, please see the last assertion in the test